sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -988,6 +990,45 @@ def custom_metric(x1, x2):
     assert_array_almost_equal(dist1, dist2)
 
 
+def test_valid_brute_metric_for_auto_algorithm():
+    X = rng.rand(12, 12)
+    Xcsr = csr_matrix(X)
+
+    # Metric which don't required any additional parameter
```

Comment:
I find the comment a bit misleading. I would rename the variable and say ``only test metrics that don't require additional arguments``.
And maybe assert that some strange metric is in ``VALID_METRICS['brute']``.